website/source/docs/platform/k8s/service-sync.html.md

```diff
@@ -0,0 +1,274 @@
+
+**If a conflicting service name exists in Consul,** the sync program will
+register additional instances to that same service. Therefore, services inside
+and outside of Kubernetes should have different names unless you want either
+side to potentially connect. This default behavior also enables gracefully
+transitioning a service from outside of K8S to inside, and vice versa.
+
+### Service Ports
+
+When syncing the Kubernetes service to Consul, the Consul service port will be
+the first defined port in the service. Additionally, all ports will be
+registered in the service instance metadata with the key "port-X" where X is
+the name of the port and the value is the externally accessible port.
+
+The default service port can be overridden using an annotation:
+
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: my-service
+  annotations:
+    "consul.hashicorp.com/service-port": "http"
+```
+
+The annotation value may a name of a port (recommended) or an exact port value.
+
+### Service Tags
+
+A service registered in Consul from K8S will always have the tag "k8s" added
+to it. Additional tags can be specified with a comma-separated annotation value
+as shown below. This will also automatically include the "k8s" tag which can't
+be disabled.
+
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: my-service
+  annotations:
+    "consul.hashicorp.com/service-tags": "primary,foo"
+```
+
+### Service Meta
+
+A service registered in Consul from K8S will set the `external-source` key to
+"kubernetes". This can be used by API consumers, the UI, CLI, etc. to filter
+service instances that are set in k8s. The Consul UI (in Consul 1.2.3 and later)
+will read this value to show a Kubernetes icon next to all externally
+registered services.
+
+Additional metadata can be specified using annotations. The "KEY" below can be
+set to any key. This allows setting multiple meta values:
+
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: my-service
+  annotations:
+    "consul.hashicorp.com/service-meta-KEY": "value"
+```
+
+## Consul to Kubernetes
+
+This syncs Consul services into first-class Kubernetes services.
+Each Consul service is synced to an
+[ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
+service in Kubernetes. The external name is configured to be the Consul
+DNS entry.
+
+This enables external services to be discovered using native Kubernetes
+tooling. This can be used to ease software migration into or out of Kubernetes,
+across platforms, to and from hosted services, and more.
+
+-> **Requires Consul DNS in Kubernetes:** This feature requires that
+[Consul DNS](/docs/platform/k8s/dns.html) is configured within Kubernetes.
+In the future we hope to remove this requirement by syncing the instance
+addresses directly into service endpoints.
+
+### Sync Enable/Disable
+
+All Consul services visible with the given Consul ACL token to the sync process
+will be synced to Kubernetes.
```

Comment:
"All Consul services visible with the given Consul ACL token to the sync process will be synced to Kubernetes."

I struggled to understand this sentence. Is this equivalent to "All Consul services visible to the sync process based on its given Consul ACL token will be synced to Kubernetes"?